symplyphysics/docs/printer_code.py

```diff
@@ -245,9 +245,13 @@ def print_row(row: int) -> str:
         return "[" + ", ".join(parts) +"]"
 
     def _print_Average(self, expr: Any) -> str:
+        # The `wrap_code` field is ignored due to being unused.
```

Comment:
Move these notes to the related class. Noone will look into printing code to find out about this issue.